ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml

```diff
@@ -96,5 +96,9 @@ ref:
     default: ""
     documentation: |-
       Allow users to make S3 deletion optional
+  - name: USER_PROVISIONED_DNS
+    default: ""
+    documentation: |-
+      Allow users to select the user provisioned dns option for AWS installations during configuration.
```

Comment:
Can you add what are the accepted values for this field? It can be confused because we use `"yes"` in the step but set `Enabled` in the `install-config.yaml` field.